recipe/meta.yaml

```diff
@@ -1,22 +1,22 @@
 {% set name = "python-flint" %}
-{% set version = "0.6.0" %}
+{% set version = "0.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: f829e00774534891b38de41bc511cf6c7d6d216544a6a84b92d9e1f159de0878
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name | replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 488352f6c37933bc44558502f0bfe7d0339a6cc07e97d2a1db4afc7660fa8022
   patches:
     # Make flint.h use DLL build
     - fix_win_build.diff
 
 build:
-  number: 2
+  number: 0
+  skip: true  # [py<311]
   script:
-    - set "INCLUDE=%INCLUDE%;%LIBRARY_INC%\\flint"  # [win]
-    - {{ PYTHON }} -m pip install . --global-option=build_ext --global-option=-DMSC_USE_DLL -vvv    # [win]
+    - {{ PYTHON }} -m pip install . --global-option=build_ext --global-option=/experimental:c11atomics --global-option=build_ext --global-option=/std:c11 -vvv    # [win]
```

Comment:
cc @oscarbenjamin

Reply:
The pip help says:

>   --global-option <options>   Extra global options to be supplied to the setup.py call before the install or bdist_wheel command.

Is that applicable when we are not using setup.py?

I think `--config-settings=setup-args=` is the way to pass command line arguments to meson from pip through meson-python:
https://github.com/flintlib/python-flint/blob/117065e51d78e54050ff3a39a37912b0a2cb8e34/.github/workflows/buildwheel.yml#L194